app/models/classification.rb

```diff
@@ -509,7 +509,9 @@ def self.add_entries_from_hash(cat, entries)
 
   def validate_uniqueness_on_tag_name
     tag_name = Classification.name2tag(name, parent, ns)
-    exist_scope = Classification.includes(:tag).where(:tags => {:name => tag_name}).merge(Tag.in_region(region_id))
+    exist_scope = Classification.default_scoped.includes(:tag)
```

Comment:
Minor, but if you are breaking up into lines by dotted calls, can you move the `.includes` to its own line too?

Reply:
Sure.  If consistency is more preferred in this case, that works for me.

Reply:
Fixed.